Source/DataTypes/SvgOrientConverter.cs

```diff
@@ -22,6 +22,8 @@ public override object ConvertFrom(ITypeDescriptorContext context, System.Global
 			{
 				case "auto":
 					return (new SvgOrient(true));
+                case "auto-start-reverse":
```

Comment:
Another formatting issue, but in this case, it's because I used spaces and not tabs!